concepts/05 UI Components/Toast/00 Getting Started with Toast/05 Display a Toast/05 Use the 'notify' Method.md

```diff
@@ -0,0 +1,265 @@
+Call the **notify** method to display a Toast. This method can accept two slightly different sets of arguments. 
+
+A simpler syntax is [notify(message, type, displayTime)](/Documentation/ApiReference/Common/Utils/ui/#notifymessage_type_displayTime). You specify the [message](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#message), [type](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#type), and [displayTime](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#displayTime).
+
+To specify additional Toast properties, call the [notify(options, type, displayTime)](/Documentation/ApiReference/Common/Utils/ui/#notifyoptions_type_displayTime) method and pass an object as the first argument. The example below uses this syntax.
+
+You can specify one of the four predefined types of notifications, depending on the mood of the message:
```

Comment:
`depending on the mood` Again, very unusual choice of words for a technical doc. How do define or quantify these 'moods' you refer to?

Reply:
Would it be okay if I change "depending on the mood of the message" to just "depending on the message"?

Reply:
Yes, that would work